File: stress/t/val_flt.c
#include <stdio.h>
#include <math.h>
int main(void){
  long i,n=50; double acc,x=1.5,t; double a[64];
  for(i=0;i<n;i++){ a[i]=(double)i*0.5+1.0; }
  acc=0.0;
  for(i=0;i<n;i++){ t=a[i]*a[i]; acc=acc+t; }
  acc=sqrt(acc);
  printf("acc=%f\n",acc);
  return 0;
}

File: stress/t/val_flt.f
      PROGRAM VALFLT
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      INTEGER*8 I,N
      DIMENSION A(0:63)
      N=50
      X=1.5D0
      DO 10 I=0,N-1
        A(I)=DBLE(I)*0.5D0+1.0D0
   10 CONTINUE
      ACC=0.0D0
      DO 20 I=0,N-1
        T=A(I)*A(I)
        ACC=ACC+T
   20 CONTINUE
      ACC=SQRT(ACC)
      WRITE(*,'(A,F0.6)') 'acc=',ACC
      END
